src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -73,6 +73,16 @@ public boolean isLoggedIn() {
     return this.isLoggedIn;
   }
 
+  public boolean isModeratorOfOrganization(long organizationId) {
+    for(Entity entity : this.moderatingOrgs) {
```

Comment:
I think you should call this.getModeratingOrgs() here, instead of this.moderatingOrgs, otherwise the field may not be populated yet. Can you add a comment on the field too (or make it private?)

Reply:
make getModeratingOrgs private?

Reply:
I meant this.moderatingOrgs (not getModeratingOrgs()). The idea is that way all callers are forced to call getModeratingOrgs; which will ensure they don't make the same mistake. Up to you.

Reply:
okay I added a comment above moderating orgs to let users know the getModeratingOrgs should be called for access